java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4790_java.html

```diff
@@ -151,13 +151,121 @@ <h2>Questionable Code Example</h2>
     }
 }
 </pre>
-<h2>See</h2>
-<ul>
-  <li> <a href="https://cwe.mitre.org/data/definitions/916.html">MITRE, CWE-916</a> - Use of Password Hash With Insufficient Computational Effort
-  </li>
-  <li> <a href="http://cwe.mitre.org/data/definitions/759.html">MITRE, CWE-759</a> - Use of a One-Way Hash without a Salt </li>
-  <li> <a href="http://cwe.mitre.org/data/definitions/760.html">MITRE, CWE-760</a> - Use of a One-Way Hash with a Predictable Salt </li>
-  <li> OWASP Top 10 2017 Category A3 - Sensitive Data Exposure </li>
-  <li> <a href="https://www.sans.org/top25-software-errors/#cat3">SANS Top 25</a> - Porous Defenses </li>
-</ul>
+<h2>Questionable Code Example</h2>
```

Comment:
Looks like `questionable code example` section is duplicated here

Reply:
good catch, fixed